Modules/Tokens/test/Tokens.API.Tests.Integration/StepDefinitions/TokensApiStepDefinitions.cs

```diff
@@ -160,10 +160,10 @@ public async Task WhenAPOSTRequestIsSentToTheTokensEndpointWith(Table table)
             switch (requestConfiguration.Content)
             {
                 case var c when c.Contains("<tomorrow>"):
-                    requestConfiguration.Content = requestConfiguration.Content.Replace("<tomorrow>", DateTime.Now.AddDays(1).ToString("dd-MM-yyyy"));
+                    requestConfiguration.Content = requestConfiguration.Content.Replace("<tomorrow>", DateTime.Now.AddDays(1).ToString("MM-dd-yyyy"));
```

Comment:
Maybe you should create two properties `TomorrowAsString` and `YesterdayAsString`, so that you don't need to duplicate the `DateTime.Now.AddDays(1).ToString("MM-dd-yyyy")` statement over and over again.